### DISCUSSION TITLE
Is it possible to go to parent folder in neotree browse?

### DISCUSSION THREAD

**Discussion by Demianeen** [nvim-neo-tree/neo-tree.nvim#1308] [Q&A] [answered]
Sometimes the folder is too big to just go through it easily (node modules for example) and it would be nice to create some action that will go to the parent folder in one keypress. Is it possible to change nvim focus to specific folder?

**Comment by cseickel** [comment, marked as answer]
The `h` command in this example will do that:

https://github.com/nvim-neo-tree/neo-tree.nvim/wiki/Tips#navigation-with-hjkl

**Reply by miversen33** [reply]
I had no idea that was a thing, thats so clever. I wonder how difficult it would be to use this as a global mapping instead of just for filesystem. Netman would more or less be able to respect this by just calling`node:expand()` or `node:collapse()` in the toggle (though I would have to add a bit of reconciliation logic on navigate).

**Reply by cseickel** [reply]
I don't see why it wouldn't work as a global mapping.
